executor_manager/routers/routers.py

```
@@ -516,9 +583,30 @@ async def receive_tasks(request: TasksRequest, http_request: Request):
+            tasks_data = [task.dict() for task in request.tasks]
+            enqueued = queue_service.enqueue_tasks(tasks_data)
+
+            logger.info(
+                f"Push mode: enqueued {enqueued}/{len(request.tasks)} tasks to "
+                f"pool '{service_pool}' queue '{queue_type}'"
+            )
+        else:
+            # Pull mode (default): process tasks directly
+            task_processor.process_tasks([task.dict() for task in request.tasks])
+
         return {"code": 0}
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)